test/parallel/test-bootstrap-modules.js

```diff
@@ -11,4 +11,4 @@ const list = process.moduleLoadList.slice();
 
 const assert = require('assert');
 
-assert(list.length <= 76, list);
+assert(list.length <= 77, list);
```

Comment:
This isn't acceptable for just one function. Please find another place to put it. Every file increases Node.js boot time. (Why doesn't this tests say that?)

Reply:
There is the code-cache optimization in the works to remedy that.

Reply:
Will that work in all cases, all platforms? Regardless, that is for the future, so my request still stands.

Literally just move the code to `lib/internal/util.js`... easy.